snuba/datasets/querylog_processor.py

```diff
@@ -0,0 +1,76 @@
+import simplejson as json
+from typing import Any, Mapping, Optional, Sequence, Union
+import uuid
+
+from snuba.processor import MessageProcessor, ProcessedMessage, ProcessorAction
+
+
+class QuerylogProcessor(MessageProcessor):
+    def __get_request(self, request: Mapping[str, Any]) -> str:
+        return json.dumps({k: v for k, v in sorted(request.items())})
```

Comment:
minor nit, I think you can inline this where you use it, it is one statement only and not even too long.